nixos/modules/services/networking/seafile.nix

```
@@ -282,6 +330,42 @@ in {
           fi
         '';
       };
+
+      seafdav = mkIf cfg.webdav.enable {
+        description = "Seafile WebDAV Extension";
+        wantedBy = [ "seafile.target" ];
+        partOf = [ "seafile.target" ];
+        after = [ "network.target" "seaf-server.service" ];
+        requires = [ "seaf-server.service" ];
+        restartTriggers = [ seahubSettings ];
+        environment = {
+          PYTHONPATH = "${pkgs.seafdav.pythonPath}:${pkgs.seafdav}:/etc/seafile";
```

Do we need `/etc/seafile` in pythonpath?
```suggestion
          PYTHONPATH = "${pkgs.seafdav.pythonPath}:${pkgs.seafdav}";
```

---

This is required for loading the `seahub_settings.py` which is processed by SeafDAV. From what I understand, this is handled via django settings for actual seahub, but that is not used in SeafDAV so I'm unsure whether there's a better way to do this.